Shared/HeightBar.cs

```diff
@@ -14,7 +14,7 @@ namespace HeightBar
     [BepInPlugin("HeightBar", "HeightBarX", Version)]
     public partial class HeightBar
     {
-        internal const string Version = "3.4";
+        internal const string Version = "3.5";
```

Comment:
```suggestion
        internal const string Version = "3.4";
```
Avoid changing unrelated things like version numbers in PRs, since it's likely to cause conflicts and it makes history less clear / harder to use git blame. Please create a separate PR for changes like this (for version numbers there's no point since how the version is incremented depends on what changes were made in that release overall).